package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "logster",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
I think is breaking current code and hence it major version update. Do we need a major or minor version update? @grofers/consumer_web